torchsynth/module.py

```diff
@@ -284,11 +356,10 @@ def _forward(self, note_on_duration: T) -> Signal:
         # Calculations to accommodate attack/decay phase cut by note duration.
         attack = self.p("attack")
         decay = self.p("decay")
+        self.alpha = self.p("alpha").unsqueeze(1)
```

Comment:
Why do we need unsqueeze here?

Reply:
We always need to unsqueeze parameters before applying them to signals. parameters are 1-dimensional and signals are 2-dimensional. Sometimes we just need to do stuff with the param values before unsqueezing, which is why attack and decay don't have that above -- they just get it later.

Reply:
Right but I thought squeeze and unsqueeze makes a copy. We could probably get more performance by changing squeeze and unsqueeze to views: https://github.com/turian/torchsynth/issues/249

For another PR :)